server/cmd/ptrun-server/main.go

```diff
@@ -20,10 +21,21 @@ var (
 )
 
 func main() {
-	// Switch with zap.NewProduction() when needed
-	// or even better, add a flag to switch this as needed.
-	// Example: ./ptrun-server --debug
-	logger, err := zap.NewDevelopment()
+	var debug = flag.Bool("debug", false, "enable debug mode")
+	var host = flag.String("host", "0.0.0.0", "set server host")
+	var port = flag.String("port", "8080", "set server port")
+	var path = flag.String("path", "/ws", "set server WS path")
```

Comment:
Perhaps we should use `BoolVar` & `StringVar` functions and include a common `struct` for storing our application configuration.